README.md

```diff
@@ -44,7 +59,8 @@ Go to the repository root and use one the following solutions
 - .... your own lovely web server
 
 
-
+### Basic `bpmn-visualization` usage 
+If you never integrated BPMN Visualization in your application, see the basic [tutorial](https://github.com/process-analytics/bpmn-visualization-js/blob/master/README.md#%EF%B8%8F-usage)
```

Comment:
If we point to the lib readme for that, this would be better to remove this paragraph.
Examples provided in this repository already show various way for integrating the lib, why not explaining to use them?

Reply:
I didn't want to rewrite the same paragraph, and it's avoir to update 2 repositories, if there is an updating.

I agree in a normal case, the user see the library repo first, and this one after. 
But, it's possible that a user arrives, by chance, on this one first. It is therefore necessary to be able to redirect him towards the first to know the bases.